src/commands/mod.rs

```diff
@@ -451,6 +452,9 @@ struct NewArgs {
     /// Rebase the children of the parents onto the new change
     #[arg(long, short = 'C')]
     rebase_children: bool,
+    /// Insert the commit before the given changes
+    #[arg(long, short = 'I')]
+    insert: bool,
```

Comment:
If I understand what this does correctly, how about `--before` instead of `--insert` ?

Reply:
Or `--insert-before` and `--insert-after`?

Reply:
"Rebase the children..." would still be a good docstring for `--insert-after`. For easier tab completion, I'd also make `--before` and `--after` aliases for `--insert-before` and `--insert-after`.

In fact, `--insert-after` could also be an alias of `--rebase-children`. At this moment, I like this idea, though it might be going too far.

To be clear, if anyone sees any problems with aliases, I think just having `--insert-after` and `--insert-before` would also be OK.

Reply:
I have switched the option names to `--insert-after` (short `-A`, alias `after`) and `--insert-before` (short `B`, alias `before`).